fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/MaterializedViewRewriter.java

```diff
@@ -1268,18 +1273,39 @@ private Map<Integer, Map<String, ColumnRefOperator>> getRelationIdToColumns(Colu
         return result;
     }
 
+    private boolean isCompensationComplete(ScalarOperator rewritten,
```

Comment:
isCompensationComplete is not a good name.

Reply:
fix it.